File: frontend/futedelas-app/src/api/interfaces/Partida.ts
import { Time } from "./Time";

export interface Partida {
    partida_id: number;
    time_mandante: Time;
    time_visitante: Time;
    status: string;
    data_realizacao: string;
    dataRealizacao: string;
    hora_realizacao: string
    horaRealizacao: string;
    placar: string;
    placar_mandante: string;
    placar_visitante: string;
}

File: backend/futdelas/src/main/java/org/ifb/tcc/futdelas/domain/entities/Partida.java
package org.ifb.tcc.futdelas.domain.entities;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class Partida {
        @JsonProperty("partida_id")
        private Long partidaId;
        @JsonProperty("time_mandante")
        private Time timeMandante;
        @JsonProperty("time_visitante")
        private Time timeVisitante;
        private String status;
        private String slug;
        @JsonProperty("data_realizacao")
        private String dataRealizacao;
        @JsonProperty("hora_realizacao")
        private String horaRealizacao;
        @JsonProperty("data_realizacao_iso")
        private String dataRealizacaoIso;
        @JsonProperty("_link")
        private String link;
        private String placar;
        @JsonProperty("placar_mandante")
        private Long placarMandante;
        @JsonProperty("placar_visitante")
        private Long placarVisitante;
    }